### ISSUE TITLE
$payment->date is null; should be $payment->post_date_gmt;

### ISSUE THREAD

**Issue by stillatmylinux** [easydigitaldownloads/easy-digital-downloads#7099]
### Expected behavior
Purchase date displays January 1, 1970

### Actual behavior
It should display the correct purchase date

### Steps to reproduce the behavior
When an old purchase which does not have the '_edd_completed_date' meta, a null date displays because when the block of code below runs the $payment->date is null.  It should be $payment->post_date_gmt since it's coming directly from the wp_posts table.

Version 2.9.8
/wp-content/plugins/easy-digital-downloads/includes/payments/class-edd-payment.php LINE 2118

    private function setup_completed_date() {
        $payment = get_post( $this->ID );

        if( 'pending' == $payment->post_status || 'preapproved' == $payment->post_status || 'processing' == $payment->post_status ) {
            return false; // This payment was never completed
        }
        // bad existing code
        $date = ( $date = $this->get_meta( '_edd_completed_date', true ) ) ? $date : $payment->date;
        // fixed recommended change
        $date = ( $date = $this->get_meta( '_edd_completed_date', true ) ) ? $date : $payment->post_date_gmt;

        return $date;
    }

### Information:
PHP Version: 7.2

EDD Version (or branch): 2.9.8

WordPress Version: 4.9.9

**Comment by stillatmylinux** [comment]
@pippinsplugins This may not be a bug because $data needs to be null in this line.

/includes/payments/actions.php LINE 117
```
    // Ensure this action only runs once ever
     if( empty( $completed_date ) ) {
```

**Comment by ashleyfae** [comment]
Closing this because it should be fixed in 3.0.
